registration-processor-default.properties

```diff
@@ -876,7 +877,7 @@ mosip.kernel.salt-generator.schemaName=regprc
 mosip.kernel.salt-generator.tableName=crypto_salt
 
 # List of old reg client versions with others attribute not present but currently supported for registration, this is required to take care of the backward compatibility checks
-mosip.regproc.common.before-cbeff-others-attibute.reg-client-versions=1.1.3,1.1.4,1.1.5,1.1.5.5
+mosip.regproc.common.before-cbeff-others-attibute.reg-client-versions=1.1.3,1.1.4,1.1.5
```

Comment:
We should use mosip.regproc.common.before-cbeff-others-attibute.reg-client-versions=1.1.3,1.1.4,1.1.5,1.1.5.5

Reply:
Done